pennylane/operation.py

```diff
@@ -1108,22 +1054,7 @@ def expand(self):
         tape = qml.tape.QuantumTape(do_queue=False)
 
         with tape:
-
-            try:
-                self.decomposition()
-
-            except TypeError:
-                if self.num_params == 0:
-                    self.decomposition(wires=self.wires)
-                else:
-                    self.decomposition(*self.parameters, wires=self.wires)
-
-                warnings.warn(
-                    "Operator.decomposition() is now an instance method, and no longer accepts parameters. "
-                    "Either define the static method 'compute_decomposition' instead, or use "
-                    "'self.wires' and 'self.parameters'.",
-                    UserWarning,
```

Comment:
@mariaschuld I was thinking about this before I scrolled down!

Might be good to add a separate breaking changes bullet point to the docstring, since this is also a breaking change in behaviour

Reply:
Good point!